code/api/src/modules/order/types.js

```diff
@@ -0,0 +1,25 @@
+// Imports
+import { GraphQLObjectType, GraphQLString, GraphQLInt, GraphQLList } from 'graphql' // can also use GraphQLId
+
+import { UserType } from '../user/types'
+import CrateType from  '../crate/types'
+import { ProductDeliveryType } from '../productDelivery/types'
+
+// Order type
+const OrderType = new GraphQLObjectType({
+  name: 'order',
+  description: 'Order type',
+
+  fields: () => ({
+    id: {type: GraphQLInt },
+    user: { type: UserType },
+    crate: { type: CrateType },
+    productDeliveries: { type: new GraphQLList( ProductDeliveryType )},
+    deliveryDate: { type: GraphQLString },
+    deliveryStatus: { type: GraphQLString },
+    createdAt: { type: GraphQLString },
+    updatedAt: { type: GraphQLString }
+  })
```

Comment:
graphQL object types being defined here for a graphQL schema to let it know what it needs to do when sent specific queries/mutations. This basically tells graphQL what all the possible fields are.